spec/student_spec.rb

```diff
@@ -25,11 +28,12 @@
 
     it 'has an id that defaults to `nil` on initialization' do
       expect(josh.id).to eq(nil)
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Because before each test we are trying to create a new student table. The test right after 'has an id that defaults...' tests if the student has created their own students table (and not just an empty def self.create_table end, which was previously passing the test). We are skipping the create table in the tests for that specific 'it' test. If we didn't explicitly drop the table the table that was created for the 'has an id that defaults...' would persist. So it's basically dropping the test table, telling the test table not to be created for that specific test and ensuring that the student is creating their own table. I hope explanation helps! Let me know if anything is unclear.